--- a/system/core/init/selinux.cpp
+++ b/system/core/init/selinux.cpp
@@ -75,7 +75,7 @@
 
 enum EnforcingStatus { SELINUX_PERMISSIVE, SELINUX_ENFORCING };
 
-EnforcingStatus StatusFromCmdline() {
+[[maybe_unused]] EnforcingStatus StatusFromCmdline() {
     EnforcingStatus status = SELINUX_ENFORCING;
 
     import_kernel_cmdline(false,
@@ -88,11 +88,9 @@
     return status;
 }
 
+// Always return false
 bool IsEnforcing() {
-    if (ALLOW_PERMISSIVE_SELINUX) {
-        return StatusFromCmdline() == SELINUX_ENFORCING;
-    }
-    return true;
+    return false;
 }
 
 // Forks, executes the provided program in the child, and waits for the completion in the parent.